kong/utils_test.go

```diff
@@ -1656,16 +1656,7 @@ func TestFillConsumerGroupPluginDefaults(T *testing.T) {
 	}
 }
 
-func Test_fillConfigRecord(t *testing.T) {
-	tests := []struct {
-		name     string
-		schema   gjson.Result
-		config   Configuration
-		expected Configuration
-	}{
-		{
-			name: "fills defaults for all missing fields",
-			schema: gjson.Parse(`{
+var fillConfigRecordTestSchema = `{
```

Comment:
👍 just a small nit, this could be `const`

```suggestion
const fillConfigRecordTestSchema = `{
```